modules/reference-guide/pages/endashes.adoc

```diff
@@ -0,0 +1,18 @@
+:navtitle: Endashes
+:keywords: reference, rule, endashes
+
+= En Dashes
```

Comment:
Small nit. I have not checked the other "pages". Do they use title case or sentence case?

Reply:
Cool. Sentence case I guess. thanks!